src/components/FilterSearch.tsx

```diff
@@ -51,6 +51,15 @@ export interface FilterSearchProps {
   placeholder?: string,
   /** Whether to trigger a search when an option is selected. Defaults to false. */
   searchOnSelect?: boolean,
+  /** A function which is called when a filter is selected. To use this, searchOnSelect must be false. */
+  onSelect?: (
+    currentFilter: StaticFilter | undefined,
+    setCurrentFilter: (filter: StaticFilter) => void,
```

Comment:
are all of these params necessary?

Reply:
yes, in order to be able to exactly replicate the current filter search behavior, a user would need access to all of these params